Makefile

```diff
@@ -23,7 +23,7 @@ clean:
 	rm -rf vendor
 
 test: deps
-	@go test $(BUILDFLAGS) -cover $(PACKAGES)
+	@go test $(BUILDFLAGS) -cover $(PACKAGES) ./...
```

Comment:
Yeah, it _is_ a weird case, and to be clear, not at all supported. But the change is innocuous enough, I guess, so I’ll let @runcom decide.